core/create/src/prompts/main.ts

```diff
@@ -64,14 +63,11 @@ export default async ({
       {
         name: 'deleteConfig',
         // Skip prompt if CircleCI config doesn't exist
-        type: await fs
-          .access(circleConfigPath)
-          .then(() => 'confirm' as const)
-          .catch(() => null),
+        type: originalCircleConfig ? ('confirm' as const) : null,
         // This .relative() call feels redundant at the moment. Maybe we can just
         // hard-code the config path?
```

Comment:
**praise:** nice!
**chore:** you can remove this comment now too :)

Reply:
omg I spent minutes yesterday trying to decide whether I should simplify the code by hardcoding that path without ever seeing that comment of past me also thinking it would make sense 😄